base/operators.jl

```diff
@@ -827,17 +827,28 @@ julia> [1:5;] |> x->x.^2 |> sum |> inv
 Compose functions: i.e. `(f ∘ g)(args...)` means `f(g(args...))`. The `∘` symbol can be
 entered in the Julia REPL (and most editors, appropriately configured) by typing `\\circ<tab>`.
 
+Function composition also works in prefix form: `∘(f,g)` is the same as `f ∘ g`.
+The prefix form also supports multiple function composition: `∘(f,g,h) = f ∘ g ∘ h`
+and splatting for an iterable with functions `∘(fs...)`
+
 # Examples
 ```jldoctest
 julia> map(uppercase∘first, ["apple", "banana", "carrot"])
 3-element Array{Char,1}:
  'A'
  'B'
  'C'
+
+julia> fs = (x -> 2x, x -> x/2, x -> x-1, x -> x+1); f = ∘(fs...); f(3) == 3
+true
 ```
 """
 ∘(f, g) = (x...)->f(g(x...))
 
+∘(f, g, h) = ∘(f ∘ g, h)
+∘(f, g, h, is...) = ∘(∘(f, g), h, is...)
+
+
```

Comment:
Maybe one of the empty lines can be removed: there used to be two empty lines, now there are 3. I'm not sure, but the two new definitions belong to the two-function case, so maybe there's no need for the empty line there?

Reply:
Yep.  So happy that the main issues are spacing :)